### ISSUE TITLE
Error message when there is no result for tophits

### ISSUE THREAD

**Issue by JasonQiu** [MRCIEU/ieugwasr#61]
Package throws error when the dataset has no tophits result.
@explodecomputer Could you please take a look? Possibly at https://github.com/MRCIEU/ieugwasr/commit/fb9a5f26ec1446435dae8683ae273009cb039560#diff-008c96b1d58014f529299c8d5a1ea473a359953adfb7cd8215a43af5c5cca602

E.g.
```
> ieugwasr::tophits("ebi-a-GCST90016930")
Using access token. For info on how this is used see logging_info()
ℹ The googleAuthR package is using a cached token for *****@bristol.ac.uk.
Error in tophits_query(id = id, pval = pval, clump = clump, r2 = r2, kb = kb,  : 
  There was an error, please contact the developers
```

```
GET /ebi-a-tophits/_search
{
  "query": {
    "match": {
      "gwas_id": "GCST90016930"
    }
  }
}
```
```
{
  "took" : 29,
  "timed_out" : false,
  "_shards" : {
    "total" : 75,
    "successful" : 75,
    "skipped" : 0,
    "failed" : 0
  },
  "hits" : {
    "total" : {
      "value" : 0,
      "relation" : "eq"
    },
    "max_score" : null,
    "hits" : [ ]
  }
}
```

**Comment by docZ9** [comment]
I also encountered the same problem, have you solved it？

**Comment by Jiang337** [comment]
I met the same problems like you. did you solve this?

**Comment by Jiang337** [comment]
I copied the 'ieugwasr' and 'TwoSampleMR' packages from someone else's computer to mine, and successfully resolved the issue

**Comment by JasonQiu** [comment]
Fixed in 07f55d02f4e7b5003d8c26c18a97ee560f982b4c
